clusterloader2/testing/neg/config.yaml

```diff
@@ -118,3 +118,9 @@ steps:
     Method: WaitForControlledPodsRunning
     Params:
       action: gather
+- name: Wait 15 minutes
+  measurements:
+    - Identifier: Wait
+      Method: Sleep
+      Params:
+        duration: 15m
```

Comment:
I suggest parametrizing the sleep duration value, so that separate tests can have a custom wait at the end. This will allow more accurate test configurations, and won’t require making changes to the test manifests in this repo.

Check out the CL2 params at top of the file, and add one such.
For example:
```
{{$waitAfterDeletion := DefaultParam .CL2_WAIT_AFTER_DELETION “15m”}}
...
- name: Wait after deletion
  measurements:
    - Identifier: Wait
      Method: Sleep
      Params:
        duration: {{$waitAfterDeletion}}
```

Please retest it before submitting.

Reply:
Updated, thank you!